crates/store/re_types/definitions/rerun/blueprint/archetypes/default_camera.fbs

```diff
@@ -0,0 +1,16 @@
+namespace rerun.blueprint.archetypes;
+
+
+/// Defines a default camera view.
+table DefaultCamera (
```

Comment:
Maybe more like
```suggestion
table DefaultEyeCameraPose (
```

The problem with `camera` is that...
* it gets confused with logged pinhole cameras a lot which is why we just call it `eye` in the implementation
    * `eye` alone though is probably even more confusing 
* this is not a full camera specification but just where it is. Later we'll have to add more things like camera controls (orbit vs ego vs future stuff) etc. which is then the "full specification" which will have a lot more fields